libGitWrap/Events/Private/GitEventCallbacks.cpp

```diff
@@ -213,8 +212,9 @@ namespace Git
         void CheckoutCallbacks::checkoutProgress(const char* path, size_t completed_steps, size_t total_steps, void* payload)
         {
             ICheckoutEvents* events = static_cast< ICheckoutEvents* >( payload );
+            Q_ASSERT( events );
 
-            qreal progress = 100 / total_steps * completed_steps;
+            qreal progress = total_steps ? (100 / total_steps * completed_steps) : 100;
```

Comment:
Ah forget about it! We don't need that: See GitWrap@eadcaa0d1caa99a7b5576cde52676e28093b05d4